.coveragerc_py27

```diff
@@ -6,14 +6,14 @@ timid = True
 exclude_lines =
     pragma: no cover
     pragma: py2 no cover
-    if six.PY2
-    elif six.PY2
+    if six.PY3
```

Comment:
should it really be 'six.PY3' in both places?
just curious: how does this logic works?

Reply:
it is a string match. Any code branch that starts with ```if six.PY3``` will be removed from python 2 coverage checking.